src/main/charts/bitbucket/Changelog.md

```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+
+## 1.0.0
+This is the first officially supported version of the Helm Charts.
```

Comment:
`charts` should probably be lowercase to be consistent with our other documentation.

Reply:
Hmm @tan-ro whats the go on this? I thought `Helm Charts`?